main.go

```diff
@@ -24,7 +24,7 @@ var (
 	// Version represents the version of the service that is running
 	Version string
 
-	// Errors
+	// ErrRegisterHealthCheck Errors
```

Comment:
You may want to be a bit more descriptive with the comment for this global variable?